distage/distage-core/.jvm/src/test/scala/izumi/distage/compat/ZIOResourcesTestJvm.scala

```diff
@@ -127,7 +127,7 @@ final class ZIOResourcesTestJvm extends AnyWordSpec with GivenWhenThen {
       """
         )
       )
-      assert(res.getMessage contains "could not find implicit value for parameter adapt: izumi.distage.model.definition.Lifecycle.AdaptProvider.Aux")
+      assert(res.getMessage contains "diverging implicit expansion for type cats.kernel.Order[A]")
```

Comment:
@neko-kai,  Those tests for binding fail (maybe due to new implicit instances), but I've started to retrieve this new error which is really odd.

Reply:
@VladPodilnyk Does this happen on 2.12 only or on 2.13 as well? 2.12 reports diverging expansion a lot more often than 2.13

Reply:
Ah, yeah, I see the build fails only on 2.12...

Reply:
Hmm, tbh IFF that's the case I'd just make different tests for 2.12 & 2.13.
2.12 is thankfully being phased out a lot quicker than 2.11 was, so 2.12 quirks shouldn't stop progress for us.

Reply:
Different tests? Those test fails for 2.12 & 2.13. I could only fix them  with changing an error message to `diverging implicit expansion`. So, maybe I did smth wrong with my instances...

Reply:
@VladPodilnyk That was because `catsMonoidForLifecycle` implicit was interfering with AdaptProvider implicits (it was in AdaptProvider's scope because AdaptProvider is nested inside `Lifecycle` object, so `Lifecycle` is its prefix type, and implicits there are part of its implicit scope)
I moved `AdaptProvider` to top level to stop this

Reply:
Oh, I get it, thanks for the explanation!